resources/assets/sass/app.scss

```diff
@@ -12,6 +12,7 @@
 @import 'components/date-picker';
 @import 'components/dropdown';
 @import 'components/helpers';
+@import 'components/partials.scss';
```

Comment:
I'd like to keep the theme of leaving out the extension with imports, as you can see in lines 12 - 14.